00-Login/.gitignore

```diff
@@ -57,6 +58,4 @@ buck-out/
 
 # CocoaPods
 /ios/Pods/
-
-# Auth0 Sample settings
-app/auth0-credentials.js
```

Comment:
This file is already in here, not sure why it's listed in the gitignore. Seems like something that _should_ be excluded?